fio_wrapper/trigger_fio.py

```diff
@@ -149,7 +149,8 @@ def _log_payload(self, directory, user, uuid, sample, fio_jobs_dict, fio_version
                                     (log_line_values[1]),
                                     # "nodeName": pod_details["hostname"],
                                     "data_direction": _data_direction[int(log_line_values[2])],
-                                    "offset": int(log_line_values[3])
+                                    "block_size": int(log_line_values[3]),
```

Comment:
is it critical that we parse block size?   Or is that already in the test parameters?      If not, then we could just skip it and continue to parse offset, this would work for both versions, right?

Reply:
yes it is, so we resolved to just support 1 fio version.